javaparser-core/src/main/java/com/github/javaparser/printer/DefaultPrettyPrinterVisitor.java

```diff
@@ -1243,6 +1243,10 @@ public void visit(final SwitchEntry n, final Void arg) {
                     printer.print(", ");
                 }
             }
+            // `case null, default -> ...` added in JEP 441
+            if (n.getLabels().isNonEmpty() && n.isDefault()) {
+                printer.print(", default");
```

Comment:
Are you sure that the "default" case must be preceded by a comma?

Reply:
In this particular case, yes. This only handles the case where `n.getLabels()` is non-empty and the case is the `default` , which would have been illegal code before java 21, but is now allowed in the special case:
```
SwitchLabel:
  case null [, default]
```

From https://docs.oracle.com/javase/specs/jls/se21/html/jls-15.html#jls-SwitchExpression